.github/stale.yml

```diff
@@ -0,0 +1,69 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 60
+
+# Number of days of inactivity before an Issue or Pull Request with the stale label is closed.
+# Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
+daysUntilClose: 7
+
+# Only issues or pull requests with all of these labels are check if stale. Defaults to `[]` (disabled)
+onlyLabels: []
+
+# Issues or Pull Requests with these labels will never be considered stale. Set to `[]` to disable
+exemptLabels:
+  - pinned
+
+# Set to true to ignore issues in a project (defaults to false)
+exemptProjects: true
+
+# Set to true to ignore issues in a milestone (defaults to false)
+exemptMilestones: true
+
+# Set to true to ignore issues with an assignee (defaults to false)
+exemptAssignees: false
+
+# Label to use when marking as stale
+staleLabel: stale
+
+# Comment to post when marking as stale. Set to `false` to disable
+markComment: >
+  This issue has been automatically marked as stale because it has not had
+  recent activity. It will be closed if no further activity occurs. Thank you!
```

Comment:
IMO we still need to update the bot to define "activity" to include references from PRs and other issues.  That will probably require a code change.

Reply:
I agree it would be nice to have; but can you think of when an issue that is being worked on from a PR is not supposed to be assigned a milestone or be in a project? would it be too much to ask those triaging issues to set that info ? then this issue aren't going to happen and if it does you comment and get 60 more days or mark it as pinned and you get infinity ;)

Reply:
I try to always put my issues into projects, but projects are not universally used and most people don't have rights to create them (of course).  And I don't think milestone is really right unless someone really wants to go and constantly update those (I know I'm not going to do it).  Most people also don't have rights to assign an issue or to add labels or milestones.

This seems to leave frequently pinging the issue as the only alternative unless a maintainer decides to pin the issue for the contributor.  That's not really ideal.

This is really looking like an XY problem to me.  We have this solution (stalebot) but I don't think there's really a broad agreement as to what the problem actually is, and who we're really trying to help.  XY problems can result in two possible outcomes: infinite discussion, or an inadequate/flawed solution.